src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/GetProfileNameDialog.xaml.cs

```diff
@@ -74,12 +74,14 @@ private void GetProfileNameDialogWindow_Loaded(object sender, RoutedEventArgs e)
             // We need to schedule this to occur later after databinding has completed, otherwise
             // focus appears in the textbox, but at the start of the suggested name rather than at
             // the end.
+#pragma warning disable VSTHRD001 // Avoid legacy threading switching APIs
```

Comment:
@jmarolf The PR to add this extension method is now out for review. I expect it will be included in the 15.5 update.